src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.Tests/assets/SdkContentTests/MsftToSbSdkFiles.diff

```diff
@@ -44,6 +44,14 @@ index ------------
  ./packs/Microsoft.NETCore.App.Ref/
  ./packs/Microsoft.NETCore.App.Ref/x.y.z/
  ./packs/Microsoft.NETCore.App.Ref/x.y.z/analyzers/
+@@ ------------ @@
+ ./sdk-manifests/
+ ./sdk-manifests/x.y.z/
+ ./sdk-manifests/x.y.z/
+-./sdk-manifests/x.y.z/
```

Comment:
The msft sdk has the following versions in sdk-manifests:
- 8.0.100
- 9.0.100-preview6
- 10.0.100-alpha.1

The SB sdk has the following versions in sdk-manifests:
- 8.0.100
- 10.0.100-alpha.1

Unsure why the msft has 9.0.100-preview6 of all versions? Additionally, this 9.0.100-preview6 directory contains the following subdirectories:
- microsoft.net.sdk.android
- microsoft.net.sdk.ios
- microsoft.net.sdk.maccatalyst
- microsoft.net.sdk.macos
- microsoft.net.sdk.maui
- microsoft.net.sdk.tvos

@ViktorHofer - do you know what might be happening here?

Reply:
Not really, unfortuantely

Reply:
This happens fairly often. @marcpopMSFT can describe why that is.

Reply:
Based on the fact that this happens often, I'm going to merge this PR.